code/game/objects/effects/temporary_visuals/miscellaneous.dm

```diff
@@ -676,6 +676,15 @@
 /obj/effect/temp_visual/flesh/proc/fade_out()
 	animate(src, alpha = 0, time = (duration - 4 SECONDS))
 
+/obj/effect/temp_visual/funeral_swarm
+	name = "funeral swarm"
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "funeral_swarm"
+	layer = BELOW_MOB_LAYER
+	pixel_x = -8
```

Comment:
Why did you shift its position if it's a 32x32 image?

Reply:
Didn't catch that